Services.Test/TestConstants.Properties.dbl

```diff
@@ -12,110 +12,139 @@
 ;;*****************************************************************************
 
 import Microsoft.VisualStudio.TestTools.UnitTesting
-import Newtonsoft.Json
+import System.Text.Json
 import System.Collections.Generic
 import System.Net.Http
+import System.Threading
+import System.IO
 
 namespace Services.Test
 
-    public static partial class TestConstants
+    public sealed class TestConstants
+        private static readonly lockObject, @Object, new Object()
+
+        private static instance, @TestConstants, ^null
+        public static property Instance, @TestConstants
+            method get
+            proc
+                try
+                begin
+                    Monitor.Enter(lockObject)
+                    begin
+                        if (instance == ^null)
+                            instance = JsonSerializer.Deserialize<TestConstants>(File.ReadAllText("C:\Users\devadm\Desktop\HarmonyCore\Services.Test\TestConstants.Values.json"))
```

Comment:
I dont think you want this hard coded here

Reply:
Changed hard path to env var look up.
See d801f24c78579c0780cd6932d0323b58d16a8d0c